app/app.iml

```diff
@@ -102,39 +102,39 @@
     </content>
     <orderEntry type="jdk" jdkName="Android API 23 Platform" jdkType="Android SDK" />
```

Comment:
why are all this changes needed?

It seems that no file was added / removed only order of files, but still the files don't seem to be ordered by any order?